tools/check_copyright.py

```diff
@@ -62,9 +62,9 @@
 # Supported comment styles (Python regex)
 #
 COMMENT_PATTERN = '^(( \\*)|(;)|(\\#))'
-COMPANY_PATTERN = '(Arm|Renesas)'
+COMPANY_PATTERN = '(Arm|Renesas|Linaro)'
 COMPANY_FULL_NAME_PATTERN = \
-    '(Arm Limited and Contributors|Renesas Electronics Corporation)'
+    '(Arm Limited and Contributors|Renesas Electronics Corporation|Linaro Limited and Contributors)'
 
```

Comment:
We would like to limit the line to 80 characters as mentioned in the code style doc